cfme/storage/object_store.py

```diff
@@ -0,0 +1,66 @@
+# -*- coding: utf-8 -*-
+
+from navmazing import NavigateToAttribute, NavigateToSibling
+
+import cfme.fixtures.pytest_selenium as sel
+from cfme.common import SummaryMixin, Taggable
+from cfme.web_ui import mixins
+from utils.appliance.implementations.ui import navigate_to, navigator, CFMENavigateStep
+from utils.appliance import Navigatable
+from cfme.web_ui import toolbar as tb
+from cfme.web_ui import Quadicon, match_location
+from utils import version
+from functools import partial
+
+
+match_page = partial(match_location, controller='cloud_object_store_container',
+                     title='Object Stores')
+
+
+class ObjectStore(Taggable, SummaryMixin, Navigatable):
+    """ Automate Model page of Cloud Object Stores
+
+    Args:
+        name: Name of Object Store
+    """
+
+    def __init__(self, name=None, appliance=None):
+        Navigatable.__init__(self, appliance=appliance)
+        self.name = name
+        self.quad_name = 'object_store'
+
+    def tag(self, tag, **kwargs):
```

Comment:
Please keep consistency, I think this shuld be add_tag. Did you look at whether the existing Tag Mixin would work here?